src/components/ExpensiTextInput/index.js

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
```

Comment:
Many of the same comments in the `index.ios.js` implementation are relevant in this one as well.